### ISSUE TITLE
Need clarity on computation involving first traversal 

### ISSUE THREAD

**Issue by devarshipant** [w3c/aria#676]
Example (2nd bullet under 2 B) -- http://www.w3.org/TR/accname-aam-1.1/#mapping_additional_nd_te is not clear when only element3 has innertext. Notice it says "element2 has no accessible name. The computation involves a first traversal of its aria-labelledby leading to element1, but element1's aria-labelledby is not subsequently followed."
In my testing (with updated code), element2 has an accessible name, and it is 'test1' as IDREF of element2 references element1.
Here is an example: 
```
<button type="button" id="el1" aria-labelledby="el3">test1</button>
<button type="button" id="el2" aria-labelledby="el1">test 2</button>
<button type="button" id="el3"> hello </button>
```

**Comment by joanmarie** [comment]
Would you mind filing this issue at the new repo (https://github.com/w3c/accname)? Thanks in advance and sorry for the inconvenience!

**Comment by devarshipant** [comment]
Got it. thanks!
